tap_pendo/streams.py

```diff
@@ -29,6 +29,15 @@
 # timeout request after 300 seconds
 REQUEST_TIMEOUT = 300
 
+FACTOR = 2
```

Comment:
Use BACKOFF_FACTOR

Reply:
Updated the variable name to BACKOFF_FACTOR